documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -2409,6 +2409,12 @@ hibernate.ogm.infinispan_remote.schema_file_name::
 Defines the file name of the generated Protobuf schema. Defualts to 'Hibernate_OGM_Generated_schema.proto'.
 The file name must have a valid __*.proto__ extension.
 
+hibernate.ogm.infinispan_remote.schema_override_resource::
```

Comment:
Let's use a Unix path instead of a Windows one :).

Reply:
I don't mind, but I'm afraid that 90% of the developer works on Windows

Reply:
At some point, they will deploy it on Linux, won't they? :)

I just think it's more appropriate to use a Unix path but, well, I won't fight for it if you think a Windows one is better. Note that there are also a lot of developers on MacOS these days and the Unix path works well for them too.

Reply:
I like your world, let's live there :)

Anyway Unix works for me, I don't really mind

Reply:
I would like to see some Windows examples too... but then make sure to test them ;-)

Reply:
According to the java doc of ConfigurationPropertyReader class:
*in case of URLs this string can either represent a class path element, an URL or a file system path*
so I think that we can rely on helper, used for all URL extractions from any configuration.

Reply:
What do you think about something like:
> The value of the property is of string type and it can either represent a *class path element*, an *URL* or a *file system path*.

Reply:
`The value of the property can either be a class path (example) , an URL (example) 
 or a file system path (example) .`